frontend/src/framework/storage/enums/storage-key.enum.ts

```diff
@@ -1,5 +1,6 @@
-enum StorageKey {
-    TOKEN = 'token',
-}
+const StorageKey = {
+    TOKEN: 'token',
+    IS_COLLAPSED: 'isCollapsed',
```

Comment:
Lets change it to isSidebarCollapsed because it is unknown what is collapsed at first sight